src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityLargeMiner.java

```diff
@@ -0,0 +1,628 @@
+            playerIn.sendStatusMessage(new TextComponentTranslation("gregtech.multiblock.large_miner.errorradius"), false);
+        }
+        return true;
+    }
+
+    @Override
+    public boolean hasMaintenanceMechanics() {
+        return false;
+    }
+
+    public boolean testForMax() {
+        return x.get() == Integer.MAX_VALUE && y.get() == Integer.MAX_VALUE && z.get() == Integer.MAX_VALUE;
+    }
+
+    public void initPos() {
+        if (!chunkMode) {
+            x.set(getPos().getX() - radius);
+            z.set(getPos().getZ() - radius);
+            y.set(getPos().getY() - 1);
+            startX.set(getPos().getX() - radius);
+            startZ.set(getPos().getZ() - radius);
+            startY.set(getPos().getY());
+            tempY.set(getPos().getY() - 1);
+            mineX.set(getPos().getX() - radius);
+            mineZ.set(getPos().getZ() - radius);
+            mineY.set(getPos().getY() - 1);
+        } else {
+            WorldServer world = (WorldServer) this.getWorld();
+            Chunk origin = world.getChunk(getPos());
+            ChunkPos startPos = (world.getChunk(origin.x - radius / 16, origin.z - radius / 16)).getPos();
+            x.set(startPos.getXStart());
+            z.set(startPos.getZStart());
+            y.set(getPos().getY() - 1);
+            startX.set(startPos.getXStart());
+            startZ.set(startPos.getZStart());
+            mineX.set(startPos.getXStart());
+            mineZ.set(startPos.getZStart());
+            mineY.set(getPos().getY() - 1);
+            startY.set(getPos().getY());
+            tempY.set(getPos().getY() - 1);
+        }
+    }
+
+    void resetInv() {
+        if (notifiedItemOutputList.size() > 0) {
+            invFull = false;
+            notifiedItemOutputList.clear();
+        }
+    }
+
+    public Material getMaterial() {
+        return material;
+    }
+
+    public int getTier() {
+        return this.tier;
+    }
+
+    public int getTick() {
+        return this.tick;
+    }
+
+    public int getChunkRadius() {
+        return this.chunkRadius;
+    }
+
+    public int getDrillingFluidConsumePerTick() {
+        return this.drillingFluidConsumePerTick;
+    }
+
+    public int getFortune() {
+        return this.fortune;
+    }
+
+    public int getRadius() {
+        return this.radius;
+    }
+
+    public String getRomanNumeralString() {
+        return this.romanNumeralString;
+    }
+
+    @Override
+    public boolean isWorkingEnabled() {
+        return this.isActive;
+    }
+
+    @Override
+    public void setWorkingEnabled(boolean isActivationAllowed) {
+        setActive(isActivationAllowed);
+    }
+
+    @Override
+    public <T> T getCapability(Capability<T> capability, EnumFacing side) {
+        if (capability == GregtechTileCapabilities.CAPABILITY_CONTROLLABLE) {
+            return GregtechTileCapabilities.CAPABILITY_CONTROLLABLE.cast(this);
+        }
+        return super.getCapability(capability, side);
+    }
```

Comment:
This is not needed. There is a check for the controllable capability inside the `super.getCapability()` call

Reply:
The check in the super method is for `CAPABILITY_MULTIBLOCK_CONTROLLER`, which is different from `CAPABILITY_CONTROLLABLE.`

Reply:
Its in MetaTileEntity, so two classes up